src/api/kube.ts

```diff
@@ -759,6 +759,41 @@ export class KubeHelper {
     }
   }
 
+  readClassStorage(yamlPath: string): V1Deployment {
+    const storageClass = this.safeLoadFromYamlFile(yamlPath) as V1Deployment
+    if (!storageClass.metadata || !storageClass.metadata.name) {
+      throw new Error(`Storage class from ${yamlPath} must have name specified`)
+    }
+    return storageClass
+  }
+
+  async storageClassExists(storageClassName: string): Promise<boolean> {
```

Comment:
`isStorageClassExists` or rename to `readStorageClass` and return <V1StorageClass | undefined>